src/color.c

```diff
@@ -1435,224 +1433,235 @@ _color_get_arraystruct(PyColor *color, void *closure)
+    }
+    return (PyObject *) _color_new_internal(Py_TYPE(obj1), rgba);
 }
 
 /**
  * color1 % color2
  */
-static PyObject*
-_color_mod (PyObject *obj1, PyObject *obj2)
+static PyObject *
+_color_mod(PyObject *obj1, PyObject *obj2)
 {
     Uint8 rgba[4] = { 0, 0, 0, 0 };
-    PyColor *color1 = (PyColor *)obj1;
-    PyColor *color2 = (PyColor *)obj2;
-    if (!PyObject_IsInstance (obj1, (PyObject *)&PyColor_Type) ||
-        !PyObject_IsInstance (obj2, (PyObject *)&PyColor_Type))   {
-        Py_INCREF (Py_NotImplemented);
+    pgColorObject *color1 = (pgColorObject *) obj1;
+    pgColorObject *color2 = (pgColorObject *) obj2;
+    if (!PyObject_IsInstance(obj1, (PyObject *) &pgColor_Type) ||
+        !PyObject_IsInstance(obj2, (PyObject *) &pgColor_Type))
+    {
+        Py_INCREF(Py_NotImplemented);
         return Py_NotImplemented;
-    }  
-    if (color2->data[0] != 0)
+    }
+    if (color2->data[0] != 0) {
         rgba[0] = color1->data[0] % color2->data[0];
-    if (color2->data[1] != 0)
+    }
+    if (color2->data[1] != 0) {
         rgba[1] = color1->data[1] % color2->data[1];
-    if (color2->data[2])
+    }
+    if (color2->data[2]) {
         rgba[2] = color1->data[2] % color2->data[2];
-    if (color2->data[3])
+    }
+    if (color2->data[3]) {
         rgba[3] = color1->data[3] % color2->data[3];
-    return (PyObject*) _color_new_internal (Py_TYPE (obj1), rgba);
+    }
+    return (PyObject *) _color_new_internal(Py_TYPE(obj1), rgba);
 }
 
 /**
  * ~color
  */
-static PyObject*
-_color_inv (PyColor *color)
+static PyObject *
+_color_inv(pgColorObject *color)
 {
     Uint8 rgba[4];
     rgba[0] = 255 - color->data[0];
     rgba[1] = 255 - color->data[1];
     rgba[2] = 255 - color->data[2];
     rgba[3] = 255 - color->data[3];
-    return (PyObject*) _color_new_internal (Py_TYPE (color), rgba);
+    return (PyObject *) _color_new_internal(Py_TYPE(color), rgba);
 }
 
 /**
  * int(color)
  */
-static PyObject*
-_color_int (PyColor *color)
+static PyObject *
+_color_int(pgColorObject *color)
 {
     Uint32 tmp = (color->data[0] << 24) + (color->data[1] << 16) +
                  (color->data[2] << 8) + color->data[3];
 #if !PY3
-    if (tmp < LONG_MAX)
-        return PyInt_FromLong ((long) tmp);
+    if (tmp < LONG_MAX) {
+        return PyInt_FromLong((long) tmp);
+    }
 #endif
-    return PyLong_FromUnsignedLong (tmp);
+    return PyLong_FromUnsignedLong(tmp);
 }
 
 /**
  * float(color)
  */
-static PyObject*
-_color_float (PyColor *color)
+static PyObject *
+_color_float(pgColorObject *color)
 {
     Uint32 tmp = ((color->data[0] << 24) +
                   (color->data[1] << 16) +
                   (color->data[2] << 8) +
                   color->data[3]);
-    return PyFloat_FromDouble ((double) tmp);
+    return PyFloat_FromDouble((double) tmp);
 }
 
 #if !PY3
 /**
  * long(color)
  */
-static PyObject*
-_color_long (PyColor *color)
+static PyObject *
+_color_long(pgColorObject *color)
```

Comment:
I guess we should prefix all functions with something. So this could be something like `_pg_color_long` ?